### PR TITLE
Update week-14 plan with 5 mins presentation

### PR DESCRIPTION
**Pull request by malvikasharan**
Sorry for the quick merge! Reopening the PR for review.

### CODE REVIEW

File: _ols-1/week14.md [hunk 1]
```diff
@@ -7,49 +7,49 @@ title: Week 14 - Meet your mentor + Group 1 - Final Presentation Rehearsal
 - [Week 14-1: Mentor meeting 6](#week-14-1--mentor-meeting-6)
 - [Week 14-2: Group 1 - Final presentation rehearsal](#week-14-2--group-1---final-presentation-rehearsal)
 
-# Week 14-1: Mentor meeting 6
+## Week 14-1: Mentor meeting 6
 
 **Date**: Week of April 20, 2020
 
 **Duration**: 30 minutes
```
Comment (yochannah) [inline _ols-1/week14.md:14]: hmmm, 30 mins or 90?

Comment (malvikasharan) [reply _ols-1/week14.md:14]: 30 mins for the mentoring calls, the cohort call is below with 90 mins. Should we mention that on the top?

**Comment by malvikasharan** [comment]
Ah the header links don't work on the website. I will update that.

File: _ols-1/week14.md (hunk 1 above)
Comment (yochannah) [reply _ols-1/week14.md:14]: nah, that was me being inattentive! Should be okay I think :)

**Pull request merged**